Fix the library's install_name.

SHRLD has to be set after LIBDIR is set.
--- configure.orig	2021-07-12 01:36:51.000000000 -0500
+++ configure	2022-01-24 14:05:30.000000000 -0600
@@ -6971,7 +6971,7 @@
       SHRLIB="libwcs.$LIBVER.dylib"
       SONAME="libwcs.$SHVER.dylib"
       SHRLD="$SHRLD -dynamiclib -single_module"
-      SHRLD="$SHRLD -compatibility_version $SHVER -current_version $LIBVER -install_name \$(SONAME)"
+      SHRLD="$SHRLD -compatibility_version $SHVER -current_version $LIBVER -install_name \$(LIBDIR)/\$(SONAME)"
       SHRLN="libwcs.dylib"
 
       case "$host_cpu" in
--- makedefs.in.orig	2021-07-12 01:36:49.000000000 -0500
+++ makedefs.in	2022-01-24 14:17:33.000000000 -0600
@@ -112,7 +112,6 @@
   SHRLIB   := @SHRLIB@
   SONAME   := @SONAME@
   SHRFLAGS := @SHRFLAGS@
-  SHRLD    := @SHRLD@
   SHRLN    := @SHRLN@
 
 # What subdirectories to build.
@@ -143,6 +142,8 @@
   PDFDIR   := $(DESTDIR)@pdfdir@
   MANDIR   := $(DESTDIR)@mandir@
 
+  SHRLD    := @SHRLD@
+
 # For putting timestamps in the build log.
   TIMER    := date +"%a %Y/%m/%d %X %z, executing on $$HOST"
 
